tests/load/test_load_sites.py

```diff
@@ -10,11 +10,29 @@
 
 def test_open_site(default_data_site_model):
     """Test the site data loader with valid data."""
-    da = open_site(default_data_site_model.file_path, default_data_site_model.metadata_file_path)
+    site_model = default_data_site_model()
+    da = open_site(site_model.file_path, site_model.metadata_file_path)
 
     assert isinstance(da, xr.DataArray)
     assert da.dims == ("time_utc", "site_id")
     assert "capacity_kwp" in da.coords
+    assert (da.capacity_kwp.data == 1).all()
+    assert "latitude" in da.coords
+    assert "longitude" in da.coords
+    assert da.shape == (49, 10)
+
+    assert len(np.unique(da.coords["site_id"])) == da.shape[1]
+
+
+def test_open_site_variable_capacity(default_data_site_model):
+    """Test the site data loader with valid data."""
+    site_model = default_data_site_model(variable_capacity=True)
+    da = open_site(site_model.file_path, site_model.metadata_file_path)
+
+    assert isinstance(da, xr.DataArray)
+    assert da.dims == ("time_utc", "site_id")
+    assert "capacity_kwp" in da.coords
+    assert (da.capacity_kwp.data == 2).all()
     assert "latitude" in da.coords
     assert "longitude" in da.coords
     assert da.shape == (49, 10)
```

Comment:
is it possible to make a test with variable capacity, and check that not all `da.capacity_kwp.data` are the same